google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITQueryTest.java

```diff
@@ -225,6 +226,23 @@ public void bindStringNull() {
     assertThat(row.isNull(0)).isTrue();
   }
 
+  @Test
+  public void bindJson() {
+    assumeFalse("Emulator does not yet support JSON", EmulatorSpannerHelper.isUsingEmulator());
+    Struct row =
+        execute(Statement.newBuilder("SELECT @v").bind("v").to(Value.json("{}")), Type.json());
+    assertThat(row.isNull(0)).isFalse();
+    assertThat(row.getJson(0)).isEqualTo("{}");
+  }
+
+  @Test
+  public void bindJsonNull() {
```

Comment:
nit: per convention we usually name the tests as `testBindJsonNull`

Reply:
this would make the JSON test names different from the other test names in this test class, is that okay?